api/spec/requests/spree/api/images_controller_spec.rb

```diff
@@ -32,6 +32,20 @@ module Spree
         end.to change(Image, :count).by(1)
       end
 
+      it 'can upload a new image from a valid URL' do
+        expect do
+          post spree.api_product_images_path(product.id), params: {
+            image: {
+              attachment: 'https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_92x30dp.png',
```

Comment:
What about using an image that belongs to Solidus, e.g. https://github.com/solidusio/brand/raw/1827e7afb7ebcf5a1fc9cf7bf6cf9d277183ef11/PNG/solidus-logo-dark.png ?

This way the test suite is safe, whatever Google decides to do with their assets.

Reply:
I like that suggestion.  I was definitely unsure about this one.   Will fix

Reply:
fixed in latest commit.